web/views.py

```
@@ -458,13 +476,57 @@ def enroll_course(request, course_slug):
+                referrer.add_referral_earnings(5)
+        except Profile.DoesNotExist:
+            pass
+
+    # Create payment intent for enrollment
+    if discount_obj:
+        # Apply the discount and mark it as used
+        payment_amount = discounted_price
+        discount_obj.mark_as_used()
+        messages.success(
+            request, 
+            f"A ${discount_amount} discount has been applied to your enrollment thanks to your social media share!"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)